test/telegram.js

```diff
@@ -20,6 +20,7 @@ if (!TOKEN) {
 const USERID = process.env.TEST_USER_ID || 777000;
 const GROUPID = process.env.TEST_GROUP_ID || -1001075450562;
 const GAME_SHORT_NAME = process.env.TEST_GAME_SHORT_NAME || 'medusalab_test';
+const PROVIDER_TOKEN = process.env.TEST_PROVIDER_TOKEN;
```

Comment:
Until we formulate a way to run a subset of the tests, depending on the available configurations, we should check if this token is provided (as we do with the `TEST_TELEGRAM_TOKEN` above), i.e.

```js
if (!PROVIDER_TOKEN) {
   throw new Error('Payment provider token not provided');
}
```